### PR TITLE
Patient Details page redesign

### LINKED ISSUE #3268: Redesign Patient UI
image

it is hard for a normal user to understand the below icons (investigation summary, shift patient, etc... ) are buttons or icons only, so we need to redesign the UI to a normal one 

**Note**: Whoever going to work on this, discuss it on EOD call or slack channel about the idea of design approach going to take and discuss with the team, finalize one and start coding it.

### LINKED ISSUE #3931: Redesign Patient Page
Here's the mockup for the Redesign. You can find it at https://www.figma.com/file/fqDMgeN2m4TCSz5KobZaKW/CoronaSafe-Network-Care?node-id=1237%3A576

image

### PR DESCRIPTION
**Pull request by Ashesh3**
Resolves #3931
Resolves #3268

Figma design followed: https://www.figma.com/file/fqDMgeN2m4TCSz5KobZaKW/CoronaSafe-Network-Care?node-id=1237%3A576

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/Patient/SampleTestCard.tsx [hunk 1]
```diff
@@ -80,94 +80,128 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
...
             UPDATE SAMPLE TEST STATUS
           </RoleButton>
           <button
-            onClick={(e) => navigate(`/sample/${itemData.id}`)}
-            className="px-4 py-2 shadow border bg-white rounded-md border-grey-500 text-sm font-semibold cursor-pointer hover:bg-gray-300 text-center w-full md:w-fit my-1"
+            onClick={(_e) => navigate(`/sample/${itemData.id}`)}
+            className="px-4 py-2 border bg-white rounded-md text-sm cursor-pointer hover:bg-gray-300 text-center w-full md:w-fit my-1 border-gray-500"
           >
             Sample Report
           </button>
```
Comment (rithviknishad) [inline src/Components/Patient/SampleTestCard.tsx]: Replace all `button` and material ui `Button`s with tailwind `ButtonV2`.

Comment (Ashesh3) [reply src/Components/Patient/SampleTestCard.tsx]: Done.

File: src/Components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -30,10 +30,11 @@ import { ErrorHelperText } from "../Common/HelperInputFields";
 import Modal from "@material-ui/core/Modal";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import ExpandLessIcon from "@material-ui/icons/ExpandLess";
```
Comment (rithviknishad) [inline src/Components/Patient/PatientHome.tsx]: We definitely need to get rid of material UI icons too :)

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Patient/PatientHome.tsx (hunk 2 above)
Comment (Ashesh3) [reply src/Components/Patient/PatientHome.tsx]: Yep! Done. :)

**Comment by Ashesh3** [comment]
This should also resolve #3268 , can you confirm that @nihal467 ?

**Comment by nihal467** [comment]
@Ashesh3 

https://deploy-preview-4010--care-net.netlify.app/facility/9a96c9d3-1ae6-4a69-ad1e-162d0e318d31/patient/2d825e30-b589-4798-aa1b-fce7fa6259fa

**Points to be worked only after discussing on the EOD**

image

- [ ] feel like so much white space is been wasted

image

- [ ] Discharged on can be shown in the first row itself rather than moving to the second row

image

- [ ] even though there is a lot of space for the tooltip, it is felt like shrinked

**Comment by nihal467** [comment]
@Ashesh3 After the EOD discussion, we have decided to do the following 

image

- [x] remove the age, which is adjacent to the gender since we are already showing the age along with the name
- [x] move the badges from the bottom of the card to the right side corner
- [x] move the blood group from the bottom to the right side, adjacent to the gender.

image

- [x] make the tooltip data in a way that, the first row shows the user name, the second row shows the user id and the third row shows the role

image

- [x] when a patient is admitted, then only show the admitted on date, and remove admitted - YES

image

- [x] when a patient is not admitted, then we should be only showing admitted - No , and remove the admitted on date from the page

**Comment by Ashesh3** [comment]
@nihal467 Required changes are made.

**Comment by nihal467** [comment]
@Ashesh3 
image

- [x] make the created and last edited, tooltip uniform like other sections, the first row shows the user name, the second row shows the user id and the third row shows the role

image

- [x] make the buttons left-aligned in the responsive view

Untitled

- [x] does changing the colour of buttons intentional or not ?

**Comment by Ashesh3** [comment]
@nihal467 Changes are done. Yes, the button change is intentional since we switched to ButtonV2.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@Ashesh3 you can use the `variant="secondary"` for the cancel button in the modal.

**Comment by Ashesh3** [comment]
> @Ashesh3 you can use the `variant="secondary"` for the cancel button in the modal.

Done!
image

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@Ashesh3 LGTM, 

https://deploy-preview-4010--care-net.netlify.app/facility/b26a1d4d-4e05-4fa2-b4d1-cd027ad686f4/patient/920c75e8-c639-4eb8-8031-9f5a301e0269

mark it as a team discussion required for final approval of UI

**Comment by nihal467** [comment]
@Ashesh3 UI is approved, @khavinshankar you can review and merge it

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**